api/v1alpha1/stage_types.go

```diff
@@ -248,11 +248,16 @@ type GitRepoUpdate struct {
 type PullRequestPromotionMechanism struct {
 	// GitHub indicates git provider is GitHub
 	GitHub *GitHubPullRequest `json:"github,omitempty" protobuf:"bytes,1,opt,name=github"`
+	// GitLab indicates git provider is Gitlab
+	Gitlab *GitLabPullRequest `json:"gitlab,omitempty" protobuf:"bytes,2,opt,name=gitlab"`
```

Comment:
Sorry, one more! Should be `GitLab *GitLabPullRequest`

Reply:
ah I missed that, thx!